app/routes/install/index.js

```diff
@@ -35,47 +35,48 @@ router.use(asyncHandler(async (req, res, next) => {
   next();
 }));
 
-router.use( queryBoolean() );
-
-// /install/:org_name/:org_key
-router.get('/:org_name/:org_key', asyncHandler(async(req, res) => {
-  const orgName = req.params.org_name;
-  const orgKey = req.params.org_key;
-  const isUpdate = req.query.update;
-
+//Ensure we have a valid api key before doing anything on the install route
+router.use(asyncHandler(async(req,res,next) => {
+  const apiKey = req.query.apiKey;
+  if(!apiKey){
+    return res.status(403).send('Missing api key.');
+  }
   const Orgs = req.db.collection('orgs');
   const org = await Orgs.findOne({
-    name: orgName,
-    apiKey: orgKey
+    apiKey: apiKey
   });
-
   if (!org) {
-    res.status(403).send(`cluster ${orgName} not found or invalid key`);
-    return;
+    return res.status(403).send('Invalid api-key.');
   }
+  req.org = org;
+  req.apiKey = apiKey;
+  next();
+}));
 
-  org.base64 = function () { 
-    return function ( value ) {
-      return Buffer.from(Mustache.render(value, org)).toString('base64');
-    };
-  };
-
-  org.isUpdate = isUpdate;
-  org.uuid = uuid();
 
-  const template = org.orgYaml;
+router.get('/inventory', asyncHandler(async(req, res, next) => {
+  const apiKey = req.apiKey;
+  var razeeapiUrl = `${req.protocol}://${req.get('host')}/api/v2`;
+  const wk_url = 'https://github.com/razee-io/watch-keeper/releases/latest/download/resource.yaml';
+  try {
+    const  inventoryBuffer = await readFile(`${__dirname}/inventory.yaml`);
```

Comment:
```suggestion
    const  inventoryBuffer = await readFile(`${__dirname}/inventory.yaml`, 'utf8');
```
should be able to specify encoding to just get a string instead of the buffer. [nodejs docs] (https://nodejs.org/api/fs.html#fs_fs_readfile_path_options_callback)

Reply:
if your npm package doesnt support the encoding option, i know `fs-extra` does